chapters/gamestructure.adoc

```diff
@@ -96,6 +96,11 @@ The <<Referee, referee>> asks the <<Robot Handler, robot handlers>> of the teams
 
 However, if both teams prefer the same color, the referee assigns the colors by chance. In this case, the teams switch the colors after the first half of the match as well as after the first half of the overtime if applicable.
 
+==== Choosing Hulls Colors
+The <<Referee, referee>> asks the <<Robot Handler, robot handlers>> for their preferred hull color (dark or bright). If teams agree, the colors are used; otherwise, the referee assigns them by chance.
```

Comment:
A reference to the other section would be nice here.

Reply:
added!